Updated Wifi-PAF section (#629)
project-chip/certification-tool: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

** The above script is just clearing the tmp files and removing the
existing networks.

[NOTE]

> It's required to run on commissionable device before running each tests

Reviewer: Is this "run on" correct? Or is it "run a"

> It's required to run on commissionable device before running each tests

Reviewer: ```suggestion
It's required to run on commissionable device before running each test
```

=== Additional Trouble Shooting Helpful Notes
* *Commissionee failed to connect to AP in running PAF commissioning*:

**Symptom**: The messages are observed on the log:
----
[1749462967.571] [3575:3576] [DL] wpa_supplicant:PropertiesChanged:key:State -> 'scanning'
[1749462969.775] [3575:3576] [DL] wpa_supplicant:PropertiesChanged:key:Scanning -> true
[1749462983.051] [3575:3576] [DL] wpa_supplicant: network scan done
[1749462983.544] [3575:3576] [DL] Network Found: (1a:27:f5:37:4e:c7) Signal:-49
[1749462983.549] [3575:3576] [DL] Network Found: Chip5 (dc:ea:e7:34:f5:5e) Signal:-51...

[1749462983.999] [3575:3576] [DL] Network Found: GRL-Public (c6:06:c3:f9:5e:ce) Signal:-61
[1749462984.004] [3575:3576] [DL] Network Found: Qualitrix (b0:19:21:73:ee:78) Signal:-77
[1749462984.008] [3575:3575] [PAF]Network resource has been unavailable for a long time
[1749462984.008] [3575:3575] [PAF]Shutdown PAF session (1, 0)
----

> **Reason**: There is a protect in SDK that it will stop sending PAF packets if device is scanning. The period is 15 seconds. If the environment is noisy, it's possible to hit the problem and failed to continue.

Reviewer: ```suggestion
**Reason**: There is a protection in SDK that makes it will stop sending PAF packets if the device is scanning. The period is of 15 seconds. If the environment is noisy, it's possible to hit the problem and fail to continue.
```

> **Solution/WAR**: Need a cleaner environment. We will add patch to make this period configurable in SDK.

Reviewer: ```suggestion
**Solution/WAR**: Need a cleaner environment. We will add a patch to make this period configurable in SDK.
```

* *Device failed to run after upgrading to v2.10 wpa_supplicant*:

> **Symptom**:Device can associate with AP, but failed at the last step if running Wi-Fi commissioning (including code-wifi, ble-wifi, wifipaf-wifi)

Reviewer: ```suggestion
**Symptom**:Device can associate with AP, but fails at the last step if running Wi-Fi commissioning (including code-wifi, ble-wifi, wifipaf-wifi)
```

> **Reason**:The problem will be observed if the `/etc/wpa_supplicant/wpa_supplicant.conf` has the ap information.

Reviewer: ```suggestion
**Reason**:The problem will be observed if the `/etc/wpa_supplicant/wpa_supplicant.conf` file has the AP information.
```
